src/java/org/apache/cassandra/utils/btree/AbstractBTreeMap.java

```diff
@@ -18,13 +18,15 @@
 
 package org.apache.cassandra.utils.btree;
 
+import java.util.AbstractCollection;
 import java.util.AbstractMap;
+import java.util.Collection;
 import java.util.Comparator;
 import java.util.Iterator;
 import java.util.Map;
 import java.util.Set;
 
-import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Iterators;
 
 public abstract class AbstractBTreeMap<K, V> extends AbstractMap<K, V>
```

Comment:
nit: We might want to consider throwing `@NotThreadSafe` on this class...not that it was before.